routes/golfer_profile.go

```diff
@@ -166,13 +166,16 @@ rows:
 
 	rows, err = db.Query(
 		`WITH ranks AS (
-		    SELECT user_id, scoring, lang,
+		    SELECT user_id, scoring, lang, SUM(points_for_lang) as total_score
```

Comment:
Did you run this? You need a trailing comma after total_score for it to be valid SQL.

And with that fixed it still doesn't do what you want as the order we fetch the rows doesn't affect the order they're displayed on the page as we use a map.

Reply:
I was going to get around to testing why the CI was failing eventually, but hopefully if this works I won't need to